src/pages/AddMembersPage/NewMemberList.tsx

```diff
@@ -313,33 +268,56 @@ const NewMemberList = ({
                 />
                 <Box css={{ color: '$currentEpochDescription', flexGrow: 1 }}>
                   <Text size="medium" color="inherit">
-                    You have added {successCount} member
-                    {successCount == 1 ? '' : 's'}
-                    !&nbsp;
-                    <Text semibold color="inherit">
-                      Share this link to get them started.
-                    </Text>
+                    You have added {newAddedCount} member
+                    {newAddedCount == 1 ? '' : 's'}!
```

Comment:
@levity I think we have a bug here when you add a previously deleted user:
it says `added 0 members` because the delete case is the update case... 

I just tested this and saw:
<img width="869" alt="Screenshot 2023-04-04 at 3 36 12 PM" src="https://user-images.githubusercontent.com/83605543/229937472-28d74f2c-36bc-4886-b6c1-e79b1e3dd3b5.png">